docroot/modules/custom/mass_alerts/tests/src/ExistingSite/EmergencyAlertsTest.php

```diff
@@ -91,7 +85,6 @@ public function testEmergencyAlertResponseSitewide() {
     $this->assertStringContainsString($alert_message_text, $page->getText());
 
     $headers = $session->getResponseHeaders();
-    $this->assertStringContainsString('max-age=60', $headers['Cache-Control'][0]);
```

Comment:
We no longer test max-age? I might have written this but I dont recall why.

Reply:
The commit is over at https://github.com/massgov/openmass/pull/1286/commits/3bf807f5bebcbf7d0b37ff4729ec2576b4a55b19

I'll try reverting this and the logout call and see if tests still pass.